tests/func/test_remote.py

```diff
@@ -430,21 +430,3 @@ def test_push_incomplete_dir(tmp_dir, dvc, mocker, local_remote):
     assert not remote_odb.exists(out.hash_info)
     assert not remote_odb.exists(file_objs[0])
     assert remote_odb.exists(file_objs[1])
-
-
-def test_upload_exists(tmp_dir, dvc, local_remote):
```

Comment:
This is testing that if the file upload succeeds or not if that file already exists in the remote. I don't think we need to test any behaviour provided by fsspec.